src/api/.rubocop.yml

```diff
@@ -25,12 +25,17 @@ Style/MultilineBlockChain:
   Exclude:
     - 'spec/**/*'
 
+
 ##################### Metrics ##################################
 
 Metrics/BlockLength:
   Exclude:
     - 'config/routes.rb'
 
+# We use the same value in haml-lint
+Metrics/LineLength:
+  Max: 150
```

Comment:
this is not having any effect: https://github.com/openSUSE/open-build-service/blob/af12274522c3864ccc4ed8824913aa7848b3aba0/src/api/.rubocop_todo.yml#L1120

It is not the first time this annoy me: https://github.com/rubocop-hq/rubocop/issues/4156 I wanted to solve it (https://github.com/rubocop-hq/rubocop/pull/5505) but it seems I didn't :see_no_evil:

Reply:
True, it's annoying... I won't set it up then...